### ISSUE TITLE
dismissable global setup not working as expected - ^0.1.4

### ISSUE THREAD

**Issue by IrvingArmenta-ML** [CaiJingLong/flutter_load_widget#3]
I setup the LoadingProvider as the latest issue was suggested: 

```
 runApp(LoadingProvider(
    themeData: LoadingThemeData(
      tapDismiss: false,
    ),
    loadingWidgetBuilder: (ctx, data) {
      return MaterialLoading();
    },
    child: Application()));
```
but it is still dismissable, as if `tapDismiss` remains `true` , could you please double test this?

**Comment by CaiJingLong** [comment]
Because the `showLoadingDialog`  's param `tapDismiss` default value is true.

I will remove the default value in next version.

![image](https://user-images.githubusercontent.com/14145407/61608231-8ba4ba80-ac84-11e9-8778-f6df133a3358.png)

**Comment by CaiJingLong** [comment]
Fixed it at version 0.1.5.
Please update the latest version.
